hack/verify-generated-completions.sh

```diff
@@ -11,7 +11,7 @@ trap "cleanup" EXIT
 
 COMPLETION_ROOT_REL="contrib/completions"
 COMPLETION_ROOT="${OS_ROOT}/${COMPLETION_ROOT_REL}"
-TMP_COMPLETION_ROOT_REL="_output/verify-generated-completions/"
+TMP_COMPLETION_ROOT_REL="_output/verify-generated-completions"
```

Comment:
I suppose I'd like a commit failing that verification so we are sure, but that can be a followup PR

Reply:
Failure happened in this PR, that's why I'm fixing it.